MergerService/src/run.cs

```diff
@@ -89,9 +89,11 @@ private string BuildPath(Source source, bool isTarget)
 
         private List<IData> BuildDataList(Source[] paths, int batchSize)
         {
+            string methodName = MethodBase.GetCurrentMethod().Name;
             using (this._activitySource.StartActivity("sources parsing"))
             {
-                List<IData> sources = new List<IData>();
+                this._logger.LogDebug($"[{methodName}] start");
```

Comment:
These logs are not allowed to be in the PR, remove all of them.

Reply:
as discussed